### PR TITLE
Add Exit Status 1 on Exception

### PR DESCRIPTION
**Pull request by m4rcelpfeifer**
As in #963 described, the bash currently gets the status code 0 if the migration fails. With this PR it will get the status code 1.

### CODE REVIEW

File: src/Phinx/Console/Command/Migrate.php [hunk 1]
```diff
@@ -113,14 +113,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        try {
+            // run the migrations
+            $start = microtime(true);
+            if ($date !== null) {
+                $this->getManager()->migrateToDateTime($environment, new \DateTime($date), $fake);
+            } else {
+                $this->getManager()->migrate($environment, $version, $fake);
+            }
+            $end = microtime(true);
+        } catch (\Throwable $e) {
```
Comment (piotr-cz) [inline src/Phinx/Console/Command/Migrate.php]: As Phinx is still compatible with [PHP v5.6](https://github.com/cakephp/phinx/blob/v0.10.6/composer.json#L28) which doesn't have a Throwable class, there should be catch for an Exception:

```suggestion
        } catch (\Exception $e) {
            $output->writeln('<error>' . $e->getMessage() . '</error>');
            return 1;
        } catch (\Throwable $e) {
```

Comment (m4rcelpfeifer) [reply src/Phinx/Console/Command/Migrate.php]: You are right, thanks.

File: src/Phinx/Console/Command/Migrate.php [hunk 2]
```diff
@@ -113,14 +113,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+            // run the migrations
+            $start = microtime(true);
+            if ($date !== null) {
+                $this->getManager()->migrateToDateTime($environment, new \DateTime($date), $fake);
+            } else {
+                $this->getManager()->migrate($environment, $version, $fake);
+            }
+            $end = microtime(true);
+        } catch (\Exception $e) {
+            $output->writeln('<error>' . $e->getMessage() . '</error>');
```
Comment (lorenzo) [inline src/Phinx/Console/Command/Migrate.php]: Not having the stack trace would be definitely annoying

Comment (lorenzo) [reply src/Phinx/Console/Command/Migrate.php]: Can you show a screenshot of how it looks like for you in the CLI?

**Comment by lorenzo** [comment]
Thanks!

**Pull request merged**